Source/WebCore/testing/Internals.idl

```diff
@@ -889,6 +889,12 @@ enum RenderingMode {
     unsigned long styleRecalcCount();
     readonly attribute unsigned long lastStyleUpdateSize;
 
+    undefined startTrackingLayoutUpdates();
+    unsigned long layoutUpdateCount();
```

Comment:
Is this different from `readonly attribute unsigned long layoutCount`? Having both is confusing.